src/OrleansPSUtils/Orleans.psd1

```diff
@@ -0,0 +1,24 @@
+﻿@{
+	RootModule = 'OrleansPSUtils.dll'
+	ModuleVersion = '1.2.3'
```

Comment:
Thanks! My mistake was the AfterBuild that was in the wrong place and ofc the endline replaces you add. Now it worked like a charm :)